intuita-webview/src/codemodList/TreeView/index.tsx

```diff
@@ -114,14 +114,14 @@ type State = Readonly<{
 type InitializerArgument = State;
 
 type Action = Readonly<{
-	kind: 'focus' | 'flip';
+	kind: 'focus' | 'flip' | 'open' | 'close';
```

Comment:
actually `flip` should flip the openness/closeness state, you probably don't need these two new states